### PR TITLE
feat(snowflake) new content for io

### PR DESCRIPTION
**Pull request by akristen**

### CODE REVIEW

**Review by pkosanam — CHANGES_REQUESTED** [review]
Requesting to kindly use the below image as a screenshot, as we are about to update the same image on the IO website too 😃

image

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            metric_type: snowflake.credit_usage_by_warehouse
+        commands: 
+        - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO credit_usage_by_warehouse.sql</var>
+    ``` 
+    </Collapser>
+</CollapserGroup>
+
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the Snowflake quickstart, you can see your critical Snowflake data in New Relic.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:150]: Once you've installed the [Snowflake quickstart](https://newrelic.com/instant-observability/snowflake), you can see your critical Snowflake data in New Relic.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    ``` 
+    </Collapser>
+</CollapserGroup>
+
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the Snowflake quickstart, you can see your critical Snowflake data in New Relic. 
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com)**, then click 
+
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:153]: 2. Sample queries to execute on NRQL to see list of integrated snowflake metrics
```
SELECT * FROM SnowflakeAccount
```
or
```
SELECT * FROM SnowflakeVirtualWarehouse
```

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+custom_attributes:
+    metric_type: snowflake.query_performance
+commands: 
+    - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO longest_queries.sql</var>
+```
+    </Collapser>
+    <Collapser
+        className="freq-link"
+        id="warehouse-usage"
+        title="Usage by warehouse"
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:115]: Credit usage by warehouse

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+event_type: SnowflakeVirtualWarehouse
+custom_attributes:
+    metric_type: snowflake.query_performance
+commands: 
+    - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO longest_queries.sql</var>
+```
+    </Collapser>
+    <Collapser
+        className="freq-link"
+        id="warehouse-usage"
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:114]: credit-usage-by-warehouse

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+        title="Usage by warehouse"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `credit_usage_by_warehouse.sql` 
+    2. Designate the file path as `newrelic-snowflake-integrations/queries`.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:120]: 2. Designate the file path as YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/queries/credit_usage_by_warehouse.sql

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    3. In `longest_queries.sql`, add the following snippet:
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:121]: 3. In `credit_usage_by_warehouse.sql`, add the following snippet:

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+        warehouse_name, 
+        sum(credits_used) as total_credits_used 
+    from 
+        snowflake.account_usage.warehouse_metering_history 
+    group by 
+        1 
+    order by 
+        2 desc;
+    ```
+    4. Add the following snippet to `flex-snowflake.yml`:
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:134]: 4. In the flex-snowflake-linux.yml file present in YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/flexConfigs/, add the following snippet:

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+    ```yml
+    - name: CreditUsageByWarehouse
+        entity: snowflake
+        event_type: SnowflakeVirtualWarehouse
+        custom_attributes:
+            metric_type: snowflake.credit_usage_by_warehouse
+        commands: 
+        - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO credit_usage_by_warehouse.sql</var>
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:143]: - run: <var>YOUR_PATH_TO_DOWNLOADED_BINARY_FILE</var> <var> YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/config.yaml</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/queries/credit_usage_by_warehouse.sql</var>

**Review by pkosanam — CHANGES_REQUESTED** [review]
Me not being a technical writer, kindly consider my grammatical mistakes if any. 😃

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,159 @@
+---
+title: Snowflake integration
+tags:
+    - Snowflake integration
+    - New Relic integrations
+metaDescription: Install our Snowflake dashboards and see your Snowflake data in New Relic.
+---
+
+import infrastructureSnowflakeDashboard from 'images/infrastructure_screenshot-full_snowflake-dashboard.png'
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:9]: are we missing a screenshot? I see two images in this branch but only one import.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+After setting up the Snowflake integration with New Relic, your data will display in dashboards like these, right out of the box. 
+</figcaption> 
+
+## Install the infrastructure agent [#infra]
+
+Before getting Snowflake data into New Relic, first install our infrastructure agent. This allows New Relic to collect data from Snowflake so performance metrics, logins, and other data appear in dashboards. 
+
+There are two methods to install the infrastructure agent:
+
+* Follow our [guided install](https://one.newrelic.com/nr1-core?account=3501165&state=ff641d8b-e722-855e-4c4a-c50c23969dc3) to instrument your system with the infrastructure agent.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:28]: we often have very little info about the difference between guided install and regular infra install. not saying you need to work on this now, as i see this in many places. something I sent to paperclypse about this: 

i keep seeing various new docs say 'use our guided install' and link directly to UI. I wonder if we might improve our guided install doc https://docs.newrelic.com/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview/ and go into more detail about why to use it and what rules/limitations are and instead link to that for more context. I can see many customers being confused around why they'd use guided install and if we send them straight to UI that doesn't really help.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+            metric_type: snowflake.credit_usage_by_warehouse
+        commands: 
+        - run: <var>YOUR_PATH_TO_DOWNLOADED_BINARY_FILE</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/config.yaml</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/credit_usage_by_warehouse.sql</var>
+    ``` 
+    </Collapser>
+</CollapserGroup>
+
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the [Snowflake quickstart](https://newrelic.com/instant-observability/snowflake), you can see your critical Snowflake data in New Relic.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:147]: maybe give a little context for how the quickstart relates to the previous instructions. If we link to it, we should make it more clear they need to follow the link. i tend to do more like 'Next, if you want to x..., then [install the Snowflake quickstart]...'

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+        - run: <var>YOUR_PATH_TO_DOWNLOADED_BINARY_FILE</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/config.yaml</var> <var>YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/credit_usage_by_warehouse.sql</var>
+    ``` 
+    </Collapser>
+</CollapserGroup>
+
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the [Snowflake quickstart](https://newrelic.com/instant-observability/snowflake), you can see your critical Snowflake data in New Relic. 
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then click **Metrics & events**.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx:149]: we could maybe be more clear that these are instructions for after they install the quickstart and want to see the dashboards. at first I thought these were instructions for installing the quickstart.

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**